session_items.py

```diff
@@ -1,77 +1,65 @@
-from flask import session
+import requests
+import os
+from copy import deepcopy
+from to_do_item import ToDoItem
 
-_DEFAULT_ITEMS = [
-    { 'id': 1, 'status': 'Not Started', 'title': 'List saved todo items' },
-    { 'id': 2, 'status': 'Not Started', 'title': 'Allow new items to be added' }
-]
+API_KEY = os.getenv('API_KEY')
+TOKEN = os.getenv('TOKEN')
+BOARD_ID = os.getenv('BOARD_ID')
+TO_DO_LIST_ID = os.getenv('TO_DO_LIST_ID')
+DONE_LIST_ID = os.getenv('DONE_LIST_ID')
 
+CREDENTIALS = { 'key': API_KEY, 'token': TOKEN }
 
-def get_items():
-    """
-    Fetches all saved items from the session.
-
-    Returns:
-        list: The list of saved items.
-    """
-    return session.get('items', _DEFAULT_ITEMS)
+GET_BOARD_URI = 'https://api.trello.com/1/boards/'
+GET_CARDS_IN_A_LIST_URI = 'https://api.trello.com/1/lists/%s/cards'
```

Comment:
Probably cleaner to use string interpolation rather than format strings (i.e. f`{BASE_URL}/lists/{DONE_LIST_ID}/cards`). Alot easier to read.